src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -32,7 +32,7 @@
     <ProjectCapability Include="UseFileGlobs"/>
     <ProjectCapability Include="EditAndContinue"/>
     <ProjectCapability Include="OpenProjectFile"/>
-    <ProjectCapability Include="GenerateNuGetPackage"/>
+    <ProjectCapability Include="Pack"/>
```

Comment:
This capability should only be for netcore projects so that all the code above to add the Package page conditionally has effect for desktop projects. So this should probably move into the SDK or better still the Pack package.